types/node/child_process.d.ts

```diff
@@ -333,13 +320,11 @@ declare module "child_process" {
         input?: string | Buffer | NodeJS.TypedArray | DataView;
         stdio?: StdioOptions;
         env?: NodeJS.ProcessEnv;
-        uid?: number;
```

Comment:
Is `extends CommonOptions` missing here? if yes, `env`, `cwd` are already included there.

Reply:
No because of `windowsHide`, I tried to dedupe a little bit here and wanted to avoid creating too many `common` interfaces.

Reply:
Sorry, don't get this. [docs](https://nodejs.org/dist/latest-v11.x/docs/api/child_process.html#child_process_child_process_execfilesync_file_args_options) tell that `uid`, `env`, and `windowsHide` are valid options for `ExecFileSync`